corehq/apps/enterprise/templates/enterprise/partials/date_range_modal.html

```diff
@@ -0,0 +1,25 @@
+{% load i18n %}
+
+<div class="modal" role="dialog" tabindex="-1" id="enterpriseFormsDaterange">
+    <div class="modal-dialog" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h5 class="modal-title">{% trans 'Customize Mobile Form Submissions Date Range' %}</h5>
+                <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
```

Comment:
Nitpick: translate the aria-label with `trans_html_attr`

Reply:
Done. I'll admit ignorance of the aria stuff, but I do notice that both my 'Cancel' and 'Apply' buttons lack them. Should I be adding aria attributes to these as well?

Reply:
I don't know either - @stephherbers @Robert-Costello do you know which buttons should/shouldn't have aria labels?

Reply:
no needed here- the screen reader can read the button with the text inside. Generally they are needed when there is information conveyed without words (so the x in the close modal)

Reply:
(to clarify here being the "cancel" and "apply" buttons)